CREATING-COMPONENT.md

```diff
@@ -0,0 +1,89 @@
+```bash
+npm install
+cd packages/core
+npm install
+cd ../..
+```
+
+### 3. Create a New Branch
+Create a new branch from develop branch to keep your changes separate from the main codebase.
+Use prefix `feat/` for feature branches and `fix/` for bug fixes.
+```bash
+git checkout -b feat/new-button
+```
+
+### 4. Develop Your Component
+Create your component in the appropriate directory. For example, if you are adding a new button component:
+```bash
+mkdir packages/core/src/components/button
+touch packages/core/src/components/button/button.tsx
+touch packages/core/src/components/button/button.scss
+touch packages/core/src/components/button/button-vars.scss
+touch packages/core/src/components/button/button.stories.tsx
+```
+Develop your component in `button.tsx`.
+Use `button.scss` for styles and `button-vars.scss` for CSS variables.
+Use `button.stories.tsx` for storybook stories.
+Import `button-vars.scss` in `global.scss`.
+Refer to the existing components for examples.
+
+### 5. Build and Run Your Local Environment
+After developing your component, you need to build and run your local environment to see your changes:
+```bash
+npm run build
+npm start
+```
+This will compile your code and start a local server with storybook where you can test your new component.
+
+### 6. Write Tests
+Write tests using Playwright for your new component to ensure it works as expected and adheres to the design system's standards.
+Keep all test files under `test` folder in the component directory. Refer to the existing tests for examples.
+
+### 7. Document Your Component
+Stencil will automatically generate a README file for your component. Make sure to link the README file to the components storybook file. If you need to provide additional documentation, add it in README.md above the lines for autogenerated documentation.
+
+### 8. Run Tests
+Make sure your code adheres to the project's coding standards and passes all tests:
+```bash
+npm run test
+```
+
+### 9. Commit Your Changes
+Once your component is ready and all checks pass, commit your changes.
+Keep in mind Tegel is using [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) as a commit convention.
+```bash
+git add .
+git commit -m "feat(button): added new component"
+```
+
+### 10. Push Your Branch
+Push your branch to the GitHub repository:
+```bash
+git push origin add-new-component
+```
+
+### 11. Create a Pull Request
+Go to the GitHub repository in your web browser. You should see a prompt to create a pull request based on your newly pushed branch. Click on "Compare & pull request", provide a description of your changes, and then submit the pull request.
+
+### 12. Review and Merge
+Wait for feedback or approval from Tegel team. Make any necessary changes based on the feedback. Once approved, merge your pull request into the main branch.
```

Comment:
Since our users don't typically have merge access, should we update this to reflect that so they know we are the ones who need to merge their branch?

Reply:
Good idea, updated!